packages/ffe-datepicker-react/src/calendar/Calendar.js

```diff
@@ -166,13 +166,27 @@ export default class Calendar extends Component {
         if (date.isLead) {
             return <LeadDate key={date.date} date={date} />;
         }
+
+        const forceFocus =
+            document.activeElement !== this.nextMonthButtonElementRef.current &&
+            document.activeElement !== this.prevMonthButtonElementRef.current
+                ? true
+                : false;
+
         return (
             <ActiveDate
                 date={date}
                 headers={`header__${this.datepickerId}__${index}`}
                 key={date.date}
                 onClick={this.mouseClick}
-                forceFocus={this.forceDateFocus}
+                forceFocus={forceFocus}
+                language={this.props.language}
+                activeRef={
+                    date.isFocus
+                        ? this.activeDateSelectedRef
+                        : this.activeDateRef
```

Comment:
Det ser ikke ut til at `ActiveDate` gjør noe med `activeRef` med mindre datoen er fokusert. Hadde det vært mulig å ikke sende inn noen ref hvis `date.isFocus` er `false`? Altså kjøre `activeRef={date.isFocus ? this.activeDateSelectedRef : undefined}`. Og så slette `activeDateRef` helt siden det ikke ser ut til at den brukes noe sted.
Og da kan du kanskje kalle `activeDateSelectedRef` for `selectedDateRef` for å få et litt kortere navn.
Og kanskje kalle `activeRef`-propen for `dateButtonRef` eller noe :shrug: